src/explore-education-statistics-admin/src/pages/release/data/components/ReorderList.module.scss

```diff
@@ -1,72 +1,50 @@
 @import '~govuk-frontend/govuk/base';
 
-.dropArea {
-  background: govuk-colour('light-grey');
-  list-style: none;
-  margin: 0 govuk-spacing(2) govuk-spacing(2) 0;
-  padding: 0;
-}
-
-.dropAreaActive {
-  outline: $govuk-focus-width solid $govuk-focus-colour;
-}
-
-.draggable {
-  background: govuk-colour('white');
-  border-bottom: 1px solid govuk-colour('dark-grey');
-  margin-bottom: 0;
-  padding: 0 0 0 govuk-spacing(7);
-  position: relative;
+.isExpandedItem {
+  background: govuk-colour('light-grey') !important;
 
   &::before {
-    color: govuk-colour('dark-grey');
-    content: '\2630';
-    cursor: grab;
-    font-size: 1.6rem;
-    left: 4px;
-    position: absolute;
-    top: 7px;
-  }
-
-  &:focus,
-  &.isDragging {
-    @include govuk-focused-text;
-  }
-
-  &.isDraggedOutside {
-    opacity: 0.7;
-  }
-}
-
-.draggable.isDisabled {
-  &::before {
-    content: none;
-  }
-}
-
-.draggable.isExpanded {
-  background: govuk-colour('light-grey');
-
-  &::before {
-    content: '\25BC';
+    content: '▼' !important;
```

Comment:
Similar to other cases - we might want to use an `aria-hidden` element instead to avoid screen readers potentially interpreting this.